File: SPACE TRIANGLE PATTERN/code5.c
#include <stdio.h>

void main (){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);


	for(int i = 1 ; i <= row ; i++){

		for(int a = 0 ; a < row - i ; a++){

			printf(" \t");
		}

		for(int j = 0; j < i ; j++){

			if(i%4==0){

				printf("%c\t",65 + j);


			}else if(i%2==0){

				printf("%c\t",64+i-j);


			}else{


				printf("%d\t",i - j);

			}

		}

		printf("\n\n");

	}


}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/9/code5.java

import java.util.Scanner;

class NestedDemo{

	public static void main(String[] args){

		Scanner s= new Scanner(System.in);
		System.out.print("Enter no of rows : ");
		int row = s.nextInt();

		int temp = 1;
		for(int i = 1; i <= row ; i++){
			

			for(int j = 0; j < row ; j++){

				if(temp % 2 == 0){
					System.out.print((temp*temp)+"		");
				}else{

					System.out.print((temp*temp*temp)+"		");
				}
				temp++;
			}

				System.out.print("\n\n");

		}

	}

}
